src/components/main-menu-items/MainMenuItem.tsx

```diff
@@ -17,7 +18,7 @@ export const MainMenuItem: React.FC<IMainMenuItem> = ({ ...props }) => {
   };
   return (
     <>
-      <SMainMenuItemWrapper>
+      <SMainMenuItemWrapper ref={props.ref}>
```

Comment:
@steiningermona this reference will be lost during rerender - please use forwardRef instead